src/natstime.c

```diff
@@ -25,7 +25,7 @@ nats_Now(void)
     struct _timeb now;
     _ftime_s(&now);
     return (((int64_t)now.time) * 1000 + now.millitm);
-#elif defined CLOCK_MONOTONIC
+#elif defined CLOCK_REALTIME
```

Comment:
Indeed. From PR: https://github.com/nats-io/nats.c/pull/63, I have this comment:
```
To be fair, this whole thing need revisiting because I used 
defined CLOCK_MONOTONIC but then only use CLOCK_REALTIME.
```